bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/CronHelper.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.scheduler;
+
+import static java.util.Objects.requireNonNull;
+import static org.apache.commons.lang.StringUtils.EMPTY;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Calendar;
+
+/**
+ * Helper class to be required for Astro Jobs to deal with CRON expressions
+ *
+ * @author Amit Kumar Mondal - Initial Contribution
+ */
+public final class CronHelper {
+
+    /** CRON Constants */
+    private static final String ANY = "*";
+    private static final String SPACE = " ";
+    private static final String SECONDS = "*";
+    private static final String EACH = "/";
+    private static final String DAYS_OF_WEEK = "?";
+
+    public static final String DAILY_MIDNIGHT = "0 0 0 * * ? *";
+
+    /** Constructor */
+    private CronHelper() {
+        throw new IllegalAccessError("Non-instantiable");
+    }
+
+    /**
+     * Returns CRON expression from the provided {@link Calendar} instance
+     *
+     * @param calendar
+     *            the {@link Calendar} instance
```

Comment:
could you please put the descriptions in the same line as the parameter throughout this PR?